drivers/gpu/drm/bridge/cdns-mhdp.c

```diff
@@ -84,6 +84,11 @@
 #define FW_STANDBY				0
```

Comment:
Can't comment on the commit desc with this crap github, so here:
- ".. still up if it is supposed..."
- "connector's"